.ci/install_go.sh

```diff
@@ -109,3 +109,17 @@ info "Install go"
 mkdir -p "${install_dest}"
 sudo tar -C "${install_dest}" -xzf "${archive}"
 popd
+
+# Make sure we use the golang version we just installed
+goroot="${install_dest}/go"
+
+[ -d "${goroot}" ] || die "failed to find expected golang path ${goroot} (from ${archive})"
+
+export GOROOT="${goroot}"
+
+gorootbin="${GOROOT}/bin"
+[ -d "${gorootbin}" ] || die "failed to find expected golang binary path ${gorootbin} (from ${archive})"
+
+export PATH="${gorootbin}:$PATH"
```

Comment:
iiuc, this script is called (nor sourced) from the jenkins script - so, these exports will be happening in a sub-shell, and not propagate up/out will they?
Maybe there needs to be a GOPATH type fix in either the jenkins script (caller) or the static script (user) ?

Reply:
Hold on - you say we need golang >= 1.10.3 - but, our versions.yaml is not fixed for that yet (as you noted on the runtime repo PR).
So, we should expect this not to work until we get golang updated, with the pending PRs.

Let me go check that PR - last I remember upstream crio got patched so we could make the version bump, let me see where we are stuck....